unpackaged/force-app/main/default/lwc/mention/mention.js

```diff
@@ -5,9 +5,11 @@ import QuillStaticResource from "@salesforce/resourceUrl/Quill";
 import { loadScript, loadStyle } from "lightning/platformResourceLoader";
 
 export default class Mention extends LightningElement {
-    @api users = [];
-    @api matchFunction = searchTerm => x => x.Name.toLowerCase().includes(searchTerm.toLowerCase());
-    @api renderItem = item => `${item.Name}`
+    @api atValues = [];
+    @api hashValues = [];
+    @api matchFunction = (searchTerm) => (x) =>
```

Comment:
@rahulgawale do we really need parentheses around variables here? Also, I don't think we need to split lines if they are no longer than 120 characters